goblet/common_cloud_actions.py

```diff
@@ -11,17 +11,20 @@
     get_default_location,
     get_credentials,
 )
+from goblet.utils import get_python_runtime
 
 log = logging.getLogger("goblet.deployer")
 log.setLevel(logging.INFO)
 
 
-def create_cloudfunction(client, req_body, config=None):
+def create_cloudfunction(client: Client, params, config={}):
```

Comment:
why are params changed here? is it different for v1 vs v2?

Reply:
in v2 you need multiple params: body and functionId (req_body was only the body param). So I changed the argument to be the full request, containing all parameters.